packngo.go

```diff
@@ -452,16 +450,19 @@ func dumpResponse(resp *http.Response) {
 	log.Printf("\n=======[RESPONSE]============\n%s\n\n", strResp)
 }
 
-func dumpRequest(req *retryablehttp.Request) {
-	o, _ := httputil.DumpRequestOut(req.Request, false)
-	strReq := string(o)
-	reg, _ := regexp.Compile(`X-Auth-Token: (\w*)`)
-	reMatches := reg.FindStringSubmatch(strReq)
-	if len(reMatches) == 2 {
-		strReq = strings.Replace(strReq, reMatches[1], strings.Repeat("-", len(reMatches[1])), 1)
+func dumpRequest(req *http.Request) {
+	r := req.Clone(context.TODO())
+	h := r.Header
+	if len(h.Get("X-Auth-Token")) != 0 {
+		h.Set("X-Auth-Token", "**REDACTED**")
```

Comment:
This is not working.

Reply:
I was outputting the original request instead of the redacted clone. fixed in d486896.